.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -7,5 +7,5 @@ contact_links:
     url: https://aka.ms/feedback/report?space=61
     about: Log product issues here
   - name: .NET Q&A forums
-    url: https://docs.microsoft.com/answers/products/dotnet
-    about: Find answers to your technical questions about .NET
+    url: /answers/products/dotnet
```

Comment:
This is not published on docs, so this file should be reverted.

Reply:
It is there, just that it's a relative link (and cross-repo). Once in master (and live) it will resolve correctly.

Reply:
Sorry, I mean that .github/ISSUE_TEMPLATE/config.yml isn't published in docs, so a link that looks like "/answers/products/dotnet" won't make any sense. Same with the missingapi.yml file.